### DISCUSSION TITLE
Render directly on Linux framebuffer?

### DISCUSSION THREAD

**Discussion by huming2207** [emilk/egui#2328] [Q&A]
Hi there, 

I'm trying to make a full-screen HMI app on some chips like Allwinner D1s or BL808, which only has 64 to 128MB RAM. I would like to know if is it possible, or any suggestions to make it possible, to render egui on Linux framebuffer? It might be a good alternative for Qt with EGLFS or Linux framebuffer, i.e. something like this: https://doc.qt.io/qt-6/embedded-linux.html

Regards,
Jackson

**Comment by rnik12** [comment]
Didn't you try Allwinner H3 or H616 ? They are available in plenty and would connect to display using HDMI.
